src/main/java/uk/ac/ebi/eva/pipeline/io/writers/VepAnnotationMongoWriter.java

```diff
@@ -60,25 +60,12 @@
 public class VepAnnotationMongoWriter extends MongoItemWriter<VariantAnnotation> {
```

Comment:
See comments on `VariantMongoWriter` about what seems to be the recommended way to implement this. The previous implementation actually seemed to follow them more than the current one.